settings.gradle.kts

```diff
@@ -0,0 +1,23 @@
+pluginManagement {
+    repositories {
+        gradlePluginPortal()
+        google()
+        mavenCentral()
+    }
+}
+dependencyResolutionManagement {
+    repositoriesMode.set(RepositoriesMode.FAIL_ON_PROJECT_REPOS)
+    repositories {
+        google()
+        mavenCentral()
+        maven(
+            "https://s01.oss.sonatype.org/content/repositories/snapshots/"
+        )
+        maven(
+            "https://s01.oss.sonatype.org/content/repositories/comwhat3words-1423"
```

Comment:
Hey @mani-w3w, any mistake on this  "https://s01.oss.sonatype.org/content/repositories/comwhat3words-1423"
?

Reply:
Ah yeah that was there already, we didn't remove when we released last time, will remove it now, nice one